lib/CorresK_Method.thy

```diff
@@ -475,7 +475,7 @@ lemmas corres_rv_lifts =
 lemmas corres_rv_prove_simple =
   corres_rv_proveT[# \<open>thin_tac _, thin_tac _\<close>, simplified]
 
-method corres_rv =
+method corresK_rv =
   (((repeat_new \<open>rule corres_rv_trivials corres_rv_lifts\<close>)?);
     ((rule corres_rv_trivials corres_rv_defers corres_rv_noops |
      (succeeds \<open>rule corres_rv_defer_left corres_rvE_R_defer_left\<close>,
```

Comment:
number of `corres_` that aren't renamed here, thoughts?

Reply:
For this specific one (`corres_rv`), I have now left the lemmas and lemma sets the same, because the `corres_rv` definition and lemmas would also still make sense for plain `corres`, but I'm renaming the method, because we'd only expect that to work in the `corresK` setup.